client/app/components/packages/deis/attached-documents.hbs

```diff
@@ -9,7 +9,7 @@
   </p>
 
   <Packages::Attachments
-    @package={{@model}}
+    @package={{@form.data}}
     @fileManager={{@model.fileManager}}
```

Comment:
What about the @model.fileManager? should it be updated to @form.data.fileManager?

Reply:
In this case, it needs to be `@ model.filemanager` because the fileManager does not relate to Validations/Changesets